CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+## How to contribute
+
+If you are interested in joining the project, then please review our [Prroject Wiki](https://github.com/hackforla/VRMS/wiki/Join-the-Team) and then review our [Project Board](https://github.com/hackforla/VRMS/projects/2) to see if anything interests you.
+
+### Committing changes overview
+
+1. Find a ticket in the [Prioritized Backlog](https://github.com/hackforla/VRMS/projects/2) and assign yourself to the ticket, and update its status as "In progress".
+
+1. Fork the repo.
+
+1. Make a branch based on the `development` branch.
+
+1. Develop the task solution.
```

Comment:
if "task solution" is common jargon among in software dev teams, fine. If not, I think this can be reworded to be clearer.